### ISSUE TITLE
can't install nvidia-vulkan-dev on linux56-rc

### ISSUE THREAD

**Issue by OneOfOne** [Tk-Glitch/PKGBUILDS#491]
```sh
┏━ oneofone@voyager ❰/n/t/nvidia-all❱ ❰master|✚3…❱
┗━● sudo dkms install nvidia/440.48.02 -k 5.6.0-rc1-1-tkg-bmq

Kernel preparation unnecessary for this kernel.  Skipping...

Building module:
cleaning build area...
'make' -j32 IGNORE_PREEMPT_RT_PRESENCE=1 NV_EXCLUDE_BUILD_MODULES='__EXCLUDE_MODULES' KERNEL_UNAME=5.6.0-rc1-1-tkg-bmq IGNORE_CC_MISMATCH='__IGNORE_CC_MISMATCH' modules.....(bad exit status: 2)
Error! Bad return status for module build on kernel: 5.6.0-rc1-1-tkg-bmq (x86_64)
Consult /var/lib/dkms/nvidia/440.48.02/build/make.log for more information.
```

[make.log](https://github.com/Tk-Glitch/PKGBUILDS/files/4187740/make.log)

**Comment by Tk-Glitch** [comment]
This is expected at this point. No publicly available Nvidia driver is compatible OOTB with 5.6 right now, and I have been a bit too busy to try and fix it myself. I'll have a look in the coming days.

**Comment by XHDR** [comment]
There's two includes missing from the NVIDIA sources. These were fixed/added in 440.59. There shouldn't be anything else that needs fixing, aside from the base 5.6-rc1 fixes.

With the includes added, the driver should build, at least.
```
diff --git a/kernel/nvidia-drm/nvidia-drm-gem-nvkms-memory.c b/kernel/nvidia-drm/nvidia-drm-gem-nvkms-memory.c
index b9cdaae..c73ed43 100644
--- a/kernel/nvidia-drm/nvidia-drm-gem-nvkms-memory.c
+++ b/kernel/nvidia-drm/nvidia-drm-gem-nvkms-memory.c
@@ -31,6 +31,8 @@
 #include <drm/drm_drv.h>
 #endif
 
+#include <linux/io.h>
+
 #include "nv-mm.h"
 
 static void __nv_drm_gem_nvkms_memory_free(struct nv_drm_gem_object *nv_gem)
diff --git a/kernel/nvidia-drm/nvidia-drm-linux.c b/kernel/nvidia-drm/nvidia-drm-linux.c
index 650b96d..1d3e658 100644
--- a/kernel/nvidia-drm/nvidia-drm-linux.c
+++ b/kernel/nvidia-drm/nvidia-drm-linux.c
@@ -35,6 +35,8 @@
 #include <drm/drmP.h>
 #endif
 
+#include <linux/vmalloc.h>
+
 #include "nv-mm.h"
 
 MODULE_PARM_DESC(

```
